# STag parameters
libraryHD: 15
errorCorrection: 7

# Camera configuration
camera_info_topic: /usb_cam/camera_info
raw_image_topic: /usb_cam/image_raw
is_compressed: true

# Node configuration

# Absolute path to JSON file
tag_config_json: /home/brennan/Documents/melodic_ws/src/stag_ros/cfg/bundle_4.json
tag_tf_prefix: stag_

# Tag corner order in JSON file:
# 4          3
#  HD########
#  #ID#  ####
#  ##  ##  ##
#  ####  ####
#  ##########
# 1          2

# Publish detected markers
show_markers: true
